org.adempiere.base/src/org/compiere/model/MSysConfig.java

```diff
@@ -137,6 +137,7 @@ public class MSysConfig extends X_AD_SysConfig
     public static final String MAIL_SEND_CREDENTIALS = "MAIL_SEND_CREDENTIALS";
     public static final String MAX_ACTIVITIES_IN_LIST = "MAX_ACTIVITIES_IN_LIST";
     public static final String MAX_RESULTS_PER_SEARCH_IN_DOCUMENT_CONTROLLER = "MAX_RESULTS_PER_SEARCH_IN_DOCUMENT_CONTROLLER";
+	public static final String MAX_ROWS_IN_LOOKUP = "MAX_ROWS_IN_LOOKUP";

```

Comment:
just a minor thing here but isn't the term "Lookup" usually associated with Search reference field in iDempiere ? I would think we usually use dropdown or combo list for Table, TableDir and List reference field.

Reply:
Thanks @hengsin - which one do you suggest?  Maybe MAX_ROWS_IN_TABLE_COMBOLIST ?

Reply:
hi @CarlosRuiz-globalqss , MAX_ROWS_IN_TABLE_COMBOLIST sounds ok

Reply:
ready